config/grafana/dashboards/load-testing/HTTP failures.json

```diff
@@ -158,31 +167,30 @@
     }
   ],
   "refresh": false,
-  "schemaVersion": 30,
+  "schemaVersion": 34,
   "style": "dark",
   "tags": [
     "home-dashboard"
   ],
   "templating": {
     "list": [
       {
-        "allValue": null,
         "current": {
-          "selected": true,
+          "selected": false,
           "text": [
             "All"
           ],
           "value": [
             "$__all"
           ]
         },
-        "datasource": "K6-InfluxDB-InfluxQL",
+        "datasource": {
```

Comment:
I'm not sure if that should change. Could cause problems when reimporting

Reply:
I don't know why that has even changed, I just adjusted the widget and exported it

Reply:
I will have a look, because looks like the dashboard format changed slighly with a new grafana release.

Reply:
I also could change the PR to contain only the changed time-frame. I guess the old format should still work, but we are not sure if the time-format will be correct

Reply:
I'll take care and reference this PR in mine, so that we can close it then